.infrastructure/app/deployment.yml

```
@@ -90,4 +90,22 @@ spec:
           secretName: app-secret
       - name: app-config-volume
         configMap:
-          name: app-config
\ No newline at end of file
+          name: app-config
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpression:
```

Incorrect name for label selector
```suggestion
              - matchExpressions:
```